server/src/module/student/grading.service.ts

```diff
@@ -134,19 +133,11 @@ export class GradingService {
      * @throws `BadRequestException` - If an error occurs during the setting process of _any_ student this exception is thrown.
      */
     async setOfMultipleStudents(dtos: Map<Student, GradingDTO>): Promise<void> {
-        const em = this.entityManager.fork({ clear: false });
-        await em.begin();
-
-        try {
-            for (const [student, dto] of dtos) {
-                const handIn = await this.getHandInFromDTO(dto);
-                await this.updateGradingOfStudent({ student, dto, em, handIn });
-            }
-            await em.commit();
-        } catch (e) {
-            await em.rollback();
-            throw new BadRequestException(e);
+        for (const [student, dto] of dtos) {
```

Comment:
Maybe this should be wrapped with a transaction? Otherwise, is it okay for one grading update to succeed while another one from the same request fails?

Reply:
to my understanding, this works as we use [implicit transactions](https://mikro-orm.io/docs/transactions#approach-1-implicitly), however, if you want to, I can switch to explicit transactions

Reply:
If this uses implicit transactions, there is no need for an explicit one, i think.